CHANGELOG.md

```diff
@@ -1,9 +1,16 @@
 # Changelog
 
-## Unreleased
-
+## [v0.8.2] 2023-12-06
 * Bug fixes:
-    * Follow Amazon EC2 IMDSv2 requirements to set `running-in-aws`
+    * Enable AWS check with IMDSv2
+    * Add region to running in AWS check
```

Comment:
There was a breaking API change here, right? We changed `.am_i_in_aws()` to `.in_region()` I think, and I think we should call that out.

Reply:
`_am_i_in_aws()` is a private method and my understanding is the `.running_on_aws` attribute is intended to be private (from @betolink in the offline `earthaccess` Slack channel). This is intended to just be an internal change, not a public-facing one.

Reply:
:heavy_check_mark: thanks, just wanted to be completely sure. Without the leading underscore, I assumed :)

Reply:
No worries -- same here